framework/discovery_agent/include/mapf/topology_discovery_agent/topology_discovery_agent.h

```diff
@@ -159,6 +159,9 @@ class DiscAgent : public TopologyDiscoveryAgentLocalDB {
         } bss_list[MAX_BSS];
     };
 
+    /**
+     * MMZ This structure is defined several times in the project
```

Comment:
topology_discovery is not built at the moment and hasn't been kept up to date.

These comments are definitely very helpful for reviving topology_discovery:
#300